README.md

```diff
@@ -1,11 +1,74 @@
-# Optimism's Contracts (V2)
+# Optimistic Ethereum Smart Contracts
 
-This package contains the various Ethereum smart contracts that make up the Layer 1 component of Optimism's Optimistic Rollup construction.
+`@eth-optimism/contracts` contains the various Solidity smart contracts used within the Optimistic Ethereum system.
+Some of these contracts are deployed on Ethereum ("Layer 1"), while others are meant to be deployed to Optimistic Ethereum ("Layer 2").
 
-## Building and Running
+Within each contract file you'll find a comment that lists:
+1. The compiler with which a contract is intended to be compiled, `solc` or `optimistic-solc`.
+2. The network upon to which the contract will be deployed, `OVM` or `EVM`.
 
-This package requires that `yarn` be installed on your machine!  Once it is, run `yarn test` to build and run tests.
+<!-- TODO: Add link to detailed contract spec here. -->
```

Comment:
http://community.optimism.io/docs/protocol/protocol.html#system-overview for now?

Reply:
Added.